tools/cli/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import commandLineArgs from 'command-line-args';
+import { Command } from './commands/Command';
+import { LoggerOptions, createLogger } from './logger';
+import { addSourcesCmd } from './sourcemaps/add-sources';
+import { processCmd } from './sourcemaps/process';
+import { uploadCmd } from './sourcemaps/upload';
+
+export interface GlobalOptions extends LoggerOptions {
+    readonly help: boolean;
+}
+
+const mainCommand = new Command<GlobalOptions>({
+    name: '',
+})
+    .subcommand(processCmd)
+    .subcommand(uploadCmd)
+    .subcommand(addSourcesCmd)
+    .option({
+        name: 'help',
+        type: Boolean,
+        alias: 'h',
+        global: true,
+        description: 'Displays this help message.',
+    })
+    .option({
+        name: 'verbose',
+        type: Boolean,
+        alias: 'v',
+        global: true,
+        multiple: true,
+        description: 'Verbosity level. -v prints info logs, -vv prints debug logs, -vvv prints ALL logs.',
+    })
+    .option({
+        name: 'quiet',
+        type: Boolean,
+        alias: 'q',
+        global: true,
+        description: 'Disables ALL logging messages.',
+    })
+    .option({
```

Comment:
What is default? Can we include what is default behavior in the log message?

Reply:
Added a default description.